cmd/kubectl-kubtest/commands/scripts/create_script.go

```diff
@@ -4,16 +4,22 @@ import (
 	"io/ioutil"
 	"os"
 
+	apiClient "github.com/kubeshop/kubtest/pkg/api/client"
+	"github.com/kubeshop/kubtest/pkg/api/kubtest"
 	"github.com/kubeshop/kubtest/pkg/ui"
 	"github.com/spf13/cobra"
 )
 
 func init() {
-	// TODO find a good way to handle short flags
 	CreateScriptsCmd.Flags().StringP("name", "n", "", "unique script name - mandatory")
 	CreateScriptsCmd.Flags().StringP("file", "f", "", "script file - will be read from stdin if not specified")
 
+	// TODO - type should be autodetected
 	CreateScriptsCmd.Flags().StringP("type", "t", "postman/collection", "script type (defaults to postman-collection)")
+
+	CreateScriptsCmd.Flags().StringP("uri", "", "", "if resource need to be loaded from URI")
```

Comment:
PersistentFlags can inject value directly into a variable. It is rather convenience so you don't have to get them `cmd.Flag("prop_name").Value.String()`

Reply:
```
var output string 

cmd.PersistentFlags().StringVarP(&output, "out", "o", "default value", "something cool to say")
```